org.gramps_project.Gramps.yml

```diff
@@ -0,0 +1,87 @@
+  - --filesystem=home
+# needs to own upstream name
+  - --own-name=org.gramps-project.Gramps
+# for gui
+  - --socket=x11
+  - --share=ipc
+#needs network access for maps
+  - --share=network
+modules:
+# oracle requires a login to get bsddb
+
+  - name: osmgpsmapDependency
+    buildsystem: autotools
+    sources:
+       - type: archive
+         url:  https://github.com/nzjrs/osm-gps-map/releases/download/1.1.0/osm-gps-map-1.1.0.tar.gz
+         sha256:  8f2ff865ed9ed9786cc5373c37b341b876958416139d0065ebb785cf88d33586
+
+  - name: exiv2Dependency
+    buildsystem: cmake
+    sources:
+      - type: archive
+        url:  https://www.exiv2.org/builds/exiv2-0.27.3-Source.tar.gz
+        sha256:  a79f5613812aa21755d578a297874fb59a85101e793edc64ec2c6bd994e3e778
+
+  - name: gexiv2Dependency
+    buildsystem: meson
+    sources:
+      - type: archive
+        url:  https://download.gnome.org/sources/gexiv2/0.12/gexiv2-0.12.1.tar.xz
+        sha256:  8aeafd59653ea88f6b78cb03780ee9fd61a2f993070c5f0d0976bed93ac2bd77
+
+  - name: PyICUDependency
+    buildsystem: simple
+    ensure-writable:
+      - /lib/python3.7/site-packages/easy-install.pth
+    build-commands:
+      - python3 setup.py install --prefix=/app
+    sources:
+      - type: archive
+        url:  https://files.pythonhosted.org/packages/31/46/fa08c8efae2951e67681ec24319f789fc1a74e2096dd74373e34c79319de/PyICU-2.6.tar.gz
+        sha256:  a9a5bf6833360f8f69e9375b91c1a7dd6e0c9157a42aee5bb7d6891804d96371
+
+  - name: GraphVizDependency
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://www2.graphviz.org/Packages/stable/portable_source/graphviz-2.44.0.tar.gz
+        sha256:  9aabd13a8018b708ab3c822de2326c19d0a52ed59f50a6b0f9318c07e2a6d93b
+
+  - name: GhostscriptDependency
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url:  https://github.com/ArtifexSoftware/ghostpdl-downloads/releases/download/gs9533/ghostscript-9.53.3.tar.gz
+        sha256:  6eaf422f26a81854a230b80fd18aaef7e8d94d661485bd2e97e695b9dce7bf7f
+
+  - name: Gramps
+    buildsystem: simple
+    ensure-writable:
+      - /lib/python3.7/site-packages/easy-install.pth
+    build-commands:
+      - python3 setup.py build -build-base=/app
+      - python3 setup.py install --prefix=/app --exec-prefix=/bin
+      - install -Dm644 /app/share/icons/gramps.png -t /app/share/icons/hicolor/48x48/apps/
+      - install -Dm644 /app/share/gramps/images/gramps.svg -t /app/share/icons/hicolor/scalable/apps/
+      - install -Dm644 org.gramps_project.gramps.desktop -t /app/share/applications/
+      - install -Dm644 org.gramps_project.Gramps.metainfo.xml -t /app/share/metainfo/
+    sources:
+      - type: archive
+        url:  https://github.com/gramps-project/gramps/archive/v5.1.3.tar.gz
+        sha256:  91d755b7e407af40ce5ebe24b29baab860bc6e2871ebcbee3f5e109c438df1e8
+      - type: file
+        path:  org.gramps_project.gramps.desktop
```

Comment:
this has inconsistent case with the file that's actually in the repo so it's not found by the builder.

Reply:
Sorry, it is fixed now.